substrate/runtime/system/src/lib.rs

```diff
@@ -117,6 +115,9 @@ decl_storage! {
 	pub ExtrinsicIndex get(extrinsic_index): b"sys:xti" => u32;
 	ExtrinsicData get(extrinsic_data): b"sys:xtd" => required map [ u32 => Vec<u8> ];
 	RandomSeed get(random_seed): b"sys:rnd" => required T::Hash;
+	ChangesTrieCreationEnabled get(changes_trie_creation_enabled): b"sys:changes_trie_creation_enabled" => default bool;
```

Comment:
everything in storage is already an option, so `bool`s are generally superfluous. probably better done as:

```
ChangesTrie => ChangesTrieConfig;
```

and then define `ChangesTrieConfig`:

```
#[derive(Encode, Decode, PartialEq, Eq, Clone)]
struct ChangesTrieConfig {
    interval: u32,
    levels: u32,
}
```